### PR TITLE
lxml: initial integration.

### PR DESCRIPTION
**Pull request by DavidKorczynski**
This integrates the lxml library.

lxml generates a substantial amount of native code during compilation

### CODE REVIEW

File: projects/lxml/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Google LLC
```
Comment (jonathanmetzman) [inline projects/lxml/Dockerfile:1]: nit: 2021.

File: projects/lxml/build.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -eu
+# Copyright 2020 Google LLC
```
Comment (jonathanmetzman) [inline projects/lxml/build.sh:2]: nit: 2021.

File: projects/lxml/fuzz_xml_parse.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python3
+
+# Copyright 2020 Google LLC
```
Comment (jonathanmetzman) [inline projects/lxml/fuzz_xml_parse.py:3]: nit: 2021

**Review by jonathanmetzman — COMMENTED** [review]
Well accept this project after upstream approves and we do a final review.

File: projects/lxml/fuzz_xml_parse.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+def TestOneInput(data):
+  fdp = atheris.FuzzedDataProvider(data)
+
+  try:
+    root = et.XML(fdp.ConsumeUnicode(sys.maxsize))
+  except et.XMLSyntaxError:
+    None
+  return
+
+atheris.Setup(sys.argv, TestOneInput, enable_python_coverage=True)
```
Comment (jonathanmetzman) [inline projects/lxml/fuzz_xml_parse.py:31]: hmm...maybe this should follow the more idiomatic way python code is written: 
```
def main():
  atheris.Setup(sys.argv, TestOneInput, enable_python_coverage=True)
  atheris.Fuzz()


if __name__ == '__main__':
  main()
```

**Review by jonathanmetzman — COMMENTED** [review]
Well accept this project after upstream approves and we do a final review.

File: projects/lxml/fuzz_xml_parse.py (hunk 4 above)
Comment (DavidKorczynski) [reply projects/lxml/fuzz_xml_parse.py:31]: Agreed - will also change this in the other python repositories I maintain.

**Comment by DavidKorczynski** [comment]
Closing this as upstream is not interested.

File: projects/lxml/Dockerfile (hunk 1 above)
Comment (DavidKorczynski) [reply projects/lxml/Dockerfile]: Thanks, fixed it to 2022

File: projects/lxml/build.sh (hunk 2 above)
Comment (DavidKorczynski) [reply projects/lxml/build.sh]: Thanks, fixed it to 2022

File: projects/lxml/fuzz_xml_parse.py (hunk 3 above)
Comment (DavidKorczynski) [reply projects/lxml/fuzz_xml_parse.py]: Thanks, fixed it to 2022

**Review by oliverchang — APPROVED** [review]

**Pull request merged**